etc/scripts/installation-functions.sh

```diff
@@ -29,6 +29,10 @@ function check_minishift {
   (hash minishift && oc whoami --show-server | grep "$(minishift ip)") >/dev/null 2>&1
 }
 
+function has_admissionapis {
+	(oc api-versions | grep admissionregistration) > /dev/null 2>&1
```

Comment:
This doesn't appear to be enough of a check, at least not on minishift, where this function returns true, yet the required `ValidatingAdmissionWebhook` and `MutatingAdmissionWebhook` stanzas are not present in the config.